src/pybind/mgr/mgr_util.py

```diff
@@ -1,5 +1,12 @@
 import os
 
+from ceph.fs.earmarking import (  # type: ignore
```

Comment:
I really think we need to stop putting type ignores in new code. However we can handle this *after* this pr.

Reply:
I think I can drop this given I've now introduced `__init__.py` for ceph/fs